go.mod

```diff
@@ -5,4 +5,5 @@ go 1.13
 require (
 	github.com/gdamore/tcell/v2 v2.0.0
 	github.com/mattn/go-runewidth v0.0.9
+	github.com/onsi/gomega v1.11.0
```

Comment:
I'm not sure if i want this testing package to be shipped to all library users, can we also test it without this package?

Reply:
Yeah, we should be able to use the in-built testing library. Let me give it a try

Reply:
This should be all set now, with the code using gomega removed I ran `go mod tidy` which removed it.